src/dlgprefreplaygaindlg.ui

```diff
@@ -202,7 +248,20 @@
       <item row="4" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
-         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not change the ReplayGain of playing tracks.</string>
+         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not apply the ReplayGain to playing tracks. </string>
+        </property>
+        <property name="wordWrap">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="0">
+       <widget class="QLabel" name="label_8">
+        <property name="text">
+         <string>The normal average ReplayGain playback level is -14 dB. This offers enough headroom for the dynamic peaks of almost all tracks. However it might be not enough for mixing or adding effects without clipping or it might sound to quiet for modern pop music that is normalized to 0 db and compressed to an average playback level of -7 dB or more.  So you may tweak the Initial Boost to solve these issues.</string>
```

Comment:
Almost yes. 

Except this: 

> Your default "loudness" will be less then in @rryan / @ywwg case and you are calling this ...

because we finally will probably have the same loudness. 

When I got @ywwg point right the majority of the tracks, in his collection are club tracks with a high dynamic range which are already close to the ReplayGain loudness. He will prefer a boost of 0 dB for both sliders. In my collection most tracks are radio edits with an higher average loudness of + 6dB compared to ReplayGain.  

An optimum setting will be in a way that all tracks are played with equal loudness, but only the higest dynamic peak of all track is at 1.0. 
This can be achieved with this PR for my and as well for @ywwg's collection (with ReplayGain enabled and disabled). 
With ReplayGain disabled it is up to the user to use the pregain. 
If a user with my collection type and Replaygain disabled starts to mix with default boost = 0 dB and pregain = 0dB he is not able to play one of @ywwg's tracks with the same loudness like the track before, because once he has cranked up the gain, the dynamic peaks are clipped. A default boost of lets say -ddB will allow him to use the pregain knob as required. 
That is a remaining issue. 

Mixxx 1.11 is optimized for my type of collection only. RJ has moved it to a more natural setting, better suitable for @ywwg tracks. 
This PR now just allows to tweak it by preferences. 

(Thank you for trying to follow my confusing explanations ;-) )